src/Pyz/Zed/EventDispatcher/EventDispatcherDependencyProvider.php

```diff
@@ -40,6 +40,44 @@ class EventDispatcherDependencyProvider extends SprykerEventDispatcherDependency
      * @return array<\Spryker\Shared\EventDispatcherExtension\Dependency\Plugin\EventDispatcherPluginInterface>
      */
     protected function getEventDispatcherPlugins(): array
+    {
```

Comment:
Why changes for src/Pyz/Zed/Console/ConsoleDependencyProvider.php is missing?

Reply:
Because the console plugin has been already there